StardewArchipelago/Locations/CodeInjections/Modded/SVE/BearShopStockModifier.cs

```diff
@@ -82,13 +83,13 @@ private StardewObject BerryIfChefsanityIsOn()
         private double BearDiscount()
         {
             var hasKnowledge = _archipelago.HasReceivedItem("Bear Knowledge");
-            var knowledgeBuff = hasKnowledge ? 0.2f: 0f;
+            var knowledgeBuff = hasKnowledge ? KNOWLEDGE_DISCOUNT : 0f;
             var applesHearts = 0;
             if (Game1.player.friendshipData.ContainsKey("Apples"))
             {
                 applesHearts = Game1.player.friendshipData["Apples"].Points / 250; // Get discount from being friends with Apples
             }
-            return INITIAL_DISCOUNT - (knowledgeBuff + applesHearts * APPLES_DISCOUNT);
+            return 1 - INITIAL_DISCOUNT - (knowledgeBuff + applesHearts * APPLES_DISCOUNT);

```

Comment:
You have reversed this value into being an actual discount, but I don't see changes in usages to not take it as a price multiplier like before

Reply:
Oh I see, the values are inverted as well. Sorry, it's been a while. Can resolve this if you want

Reply:
I'm have a hard time parsing what you mean.  If initial_discount was 0.3, and now its 0.7, then 1-0.7 = 0.3 and there's no difference other than presentation.

Reply:
Yes you are correct that's why I put the second comment that was essentially saying "nevermind I'm just dumb" lmao

Reply:
Oh and I didn't even catch that lmao